### ISSUE TITLE
Examples of the documentation should use valid URIs

### ISSUE THREAD

**Issue by PonteIneptique** [distributed-text-services/specifications#137]
Currently, some examples are using plain string, although we recommend that they use URIs (but not enforce it). I think it would be good practice to rewrite them with valid URIs.

It's a small task if people agree on this.

**Comment by monotasker** [comment]
I've made this change in the Document endpoint examples, but I've done it by simply making up a urn namespace for some of the ids. For example, with the document id "bgu;11;2029" I've made up the "urn:dc3" namespace and made the id "urn:dc3:bgu;11;2029". Does anyone think this is a problem? Should the URNs in the doc examples be actually resolvable or is it more important simply to have an example using a (fictional) valid URN?

**Comment by hcayless** [comment]
That particular example has a real URL: https://papyri.info/ddbdp/bgu;11;2029/source. I guess my question would be whether inventing a URN clarifies or obscures the example. I think if we invent URIs for things, we should make it clear in the documentation how and why we're doing so, lest people get the wrong idea.

**Comment by monotasker** [comment]
Yes, that makes sense. Thanks for the URL. That actually solves the issue anyhow.

**Comment by balmas** [comment]
I would rather see URLs than URNs in the examples.

**Comment by monotasker** [comment]
We published the resolution of this issue during the tech committee meeting on 2024-03-08

commit https://github.com/distributed-text-services/specifications/commit/a0db8ca0f5b9ef2e208baae7cacd8e2b6108685b
release https://github.com/distributed-text-services/specifications/releases/tag/1-alpha1

This is an alpha release and we are looking for feedback!
